src/reducers/node.reducer.js

```diff
@@ -1,48 +1,67 @@
 import { Map } from 'immutable';
+import invariant from 'invariant';
 
 import {
-   FLOWDESIGNER_NODE_ADD,
-   FLOWDESIGNER_NODE_MOVE,
-   FLOWDESIGNER_NODE_MOVE_END,
-   FLOWDESIGNER_NODE_UPDATE_TYPE,
-   FLOWDESIGNER_NODE_SET_ATTR,
-   FLOWDESIGNER_NODE_SET_SIZE,
-   FLOWDESIGNER_NODE_REMOVE,
+	FLOWDESIGNER_NODE_ADD,
+	FLOWDESIGNER_NODE_MOVE,
+	FLOWDESIGNER_NODE_MOVE_END,
+	FLOWDESIGNER_NODE_SET_ATTR,
+	FLOWDESIGNER_NODE_REMOVE_ATTR,
```

Comment:
what is the meaning of SET_ATTR vs REMOVE_ATTR ? I mean remove is just a set to empty right ?